individual-reflection.md

```diff
@@ -9,48 +9,65 @@ Answer the following comprehension questions **within this file.** Write your an
 ### `GET` Request Review
 
 1. Describe a GET request that your project makes, and the high-level description of what it does
-    - Answer:
+    - Answer: I used GET which is a common HTTP method, to request data from a specified resource. It was used in the recipient.rb file to get data on the user or channel that was needed from the user's perspective.
 1. What is the verb of this request?
-    - Answer:
+    - Answer: get
 1. What is the path (or the URL, or endpoint) of this request?
-    - Answer:
-1. What are the query params (the additional data sent with the request, besides the verb and the path)?
-    - Answer: 
+    - Answer: "https://slack.com/api/users.list" for users and "https://slack.com/api/users.channels" for channels.
+1. What are the query params (the additional data sent with the request, besides the verb and the path)? 
+    - Answer: I added a token that gives the request the authorization to access the data by using a specific URL depending on the recipient.
 1. What is the syntax used to make this request? (Copy and paste a code snippet here)
     - Answer:
       ```ruby
       # Copy and paste your answer below this comment
-
+        def self.get(url)
+        response = HTTParty.get(url, query: {token: ENV['SLACK_TOKEN']})
+            raise SlackAPIError, "We encountered a problem: #{response["error"]}" if response.code != 200 || response["ok"] == false
+        return response
+        end
       # Copy and paste your answer above this comment
       ```
 1. What does the program do if the response comes back with a status code of 200?
-    - Answer: 
+    - Answer: It will query the data and provide what was requested.
 1. What does the program do if the response does not come back with a status code of 200?
-    - Answer: 
+    - Answer: It will raise a SlackAPIError with a message and the error.
 
 ### `POST` Request Review
 
 If your project does not make a POST request, read through Wave 3 on the original Slack CLI, and research and answer questions 1, 2, 3, 4, 6, and 7.
 
 1. Describe a POST request that your project makes, and the high-level description of what it does
-    - Answer:
+    - Answer: The POST request was used to send data to the server we created but it can also be used to update a resource. The POST request in this program was used to send messages to a specific recipient.
 1. What is the verb of this request?
-    - Answer:
+    - Answer: send...post...#revise this
```

Comment:
Just chiming in: POST!

HTTP Requests are either of the verbs GET, POST, PATCH, PUT, DELETE... there are others, but we ignore those.